sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolHelper.cs

```diff
@@ -49,7 +49,7 @@ public static IEnumerable<INamedTypeSymbol> GetAllNamedTypes(this INamespaceSymb
             symbol?.GetAttributes().Where(a => a.AttributeClass.Is(attributeType))
                 ?? Enumerable.Empty<AttributeData>();
 
-        internal static IEnumerable<AttributeData> GetAttributes(this ISymbol symbol, ISet<KnownType> attributeTypes) =>
+        internal static IEnumerable<AttributeData> GetAttributes(this ISymbol symbol, IEnumerable<KnownType> attributeTypes) =>

```

Comment:
Changing the signature to use _IEnumerable_ so the method can be called from some of the methods in UnitTestHelper_ that don't use _ISet_.
Same for the method in _TypeHelper_.